.github/workflows/kvrocks.yaml

```
@@ -480,4 +480,4 @@ jobs:
             exit 1
           fi
       - name: Sentinel
-        run: true
+        run: true
```

Keep the empty line at the end of the file.
```suggestion
        run: true
 
```